contracts/VotiumBribesProcessor.sol

```diff
@@ -83,6 +86,7 @@ contract VotiumBribesProcessor is CowSwapSeller {
     function ragequit(IERC20 token, bool sendToGovernance) external nonReentrant {
         bool timeHasExpired = block.timestamp > lastBribeAction + MAX_MANAGER_IDLE_TIME;
         require(msg.sender == manager || timeHasExpired);
+        require(HARVEST_FORWARDER.badger_tree() == BADGER_TREE);
```

Comment:
We should only check this if `sendToGovernance` is `false`.  If it's `true` then we just send tokens directly to governance not to the tree

Reply:
Agree to run the check after the if, saves 2.2k gas when not-needed